src/main/block_table.hpp

```diff
@@ -293,7 +293,7 @@ struct screen_t
 template<class MyRef_t>
 inline Location_t extract_loc(MyRef_t obj)
 {
-    return obj->Location;
+    return (Location_t)obj->Location;
```

Comment:
Maybe use a kind of `static_cast<type>()`?

Reply:
There are many such cases, but I can switch them all to static cast if you prefer.

Reply:
Done!